dashboard/test/controllers/certificate_images_controller_test.rb

```diff
@@ -0,0 +1,32 @@
+require 'test_helper'
+require 'base64'
+
+class CertificateImagesControllerTest < ActionController::TestCase
```

Comment:
Could we add a test that includes all four parameters and also a test that includes a bogus course name?